README.md

```diff
@@ -11,39 +11,62 @@ Grafana will use the *dist/* folder by default
-- choose a name
-- set Warp10 as type
-- paste the Warp 10™ Platform `exec` endpoint URL ( do not append /api/v0/... )
+- Go to the Grafana menu (top left) > "datasources" > "add data source"
+- Choose a name
+- Set Warp10 as type
+- Paste the Warp 10™ Platform `exec` endpoint URL ( do not append /api/v0/... )
+![Global Variables](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-endpoint.png)
 
-## Add execution variables
+
+# Add execution variables
 
 You can define variables at datasource level (~ organisation level) which can be available for all dashboards. you can put tokens, constants, macros, ...
 In case of a macro definition, the variable value must start with *<%* and end with *%>*. In the query you can prepend *@* to the macro name to execute it.
 
+For example, you can store a read token here:
+![Global Variables](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-globalvar.png)
+
+
 # Make a query
 
-On a new dashboard, in a graph edition, choose your previous datasource and click *add query*.
+On a new dashboard, in a Graph visualization, click on Query icon on the left side bar, and choose Warp10 datasource.
+
+A user friendly query editor will appear. You can use the global variable you defined previously:
 
-You can write your WarpScript™ on the editor below, for beginners, you can uncheck *WarpScript editor*, a user friendly query editor will appear.
+![First query](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-simplifiedquery.png)
 
-A query is composed by 2 components, the WarpScript™ from the WarpScript™ editor and the WarpScript™ from the friendly query builder, check or uncheck the *WarpScript™ editor* will execute the corresponding WarpScript™.
+It will generate WarpScript for you. The generated WarpScript is visible in your browser console:
+![Console output](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-console.png)
 
-to graph something on Grafana you need to return some Geo Time Series™ (GTS)
+You will see at the beginning of the query some special variables set by the plugin.
+
+
+| Name          | Description                                                   | Example                    |
+|---------------|---------------------------------------------------------------|----------------------------|
+| **$end**      | Timestamp of the most recent point in the Grafana time window | 1498038153276000           |
+| **$endISO**   | *end* value in ISO-8601 format                                | '2017-06-21T09:42:33.276Z' |
+| **$start**    | Timestamp of the less recent point in the Grafana time window | 1498034553276000           |
+| **$startISO** | *start* value in ISO-8601 format                              | '2017-06-21T08:42:33.276Z' |
+| **$interval** | Difference between $end and $start                            | 3600000000                 |
```

Comment:
missing `$__interval`

Reply:
It wasn't inside the original table too... I don't understand the point of this variable. What is the use case ?

Reply:
Yes, it has been forgot, this is a Grafana defined variable
https://grafana.com/docs/grafana/latest/reference/templating/#the-interval-variable